### PR TITLE
uniffi: migrate to indy vdr

### PR DESCRIPTION
**Pull request by swaptr**

### CODE REVIEW

File: uniffi_aries_vcx/core/Cargo.toml [hunk 1]
```diff
@@ -17,9 +17,12 @@ path = "uniffi-bindgen.rs"
...
 [dependencies]
 uniffi = { version = "0.23.0", features = ["cli"] }
-aries-vcx = { path = "../../aries_vcx" }
-test_utils = {path = "../../tools/test_utils" }
-tokio = { version = "1.24.1" }
+aries-vcx = { path = "../../aries_vcx", features = [
+    "vdrtools_wallet",
+    "credx",
+] }
+test_utils = { path = "../../tools/test_utils" }
```
Comment (bobozaur) [inline uniffi_aries_vcx/core/Cargo.toml:24]: Just a heads-up that the `test-utils` was added as a full dependency to provide the `MockLedger` type. It's really meant to only exist as a `dev-dependency`, although this crate might not need it at all unless it's used for testing.

File: uniffi_aries_vcx/demo/app/src/main/java/org/hyperledger/ariesvcx/HomeScreen.kt [hunk 2]
```diff
@@ -30,7 +32,14 @@ fun HomeScreen(
 
     val scope = rememberCoroutineScope()
     val context = LocalContext.current
-
+    val file = File.createTempFile("genesis", null, context.cacheDir)
+    val genesisContent = """{"reqSignature":{},"txn":{"data":{"data":{"alias":"Node1","blskey":"4N8aUNHSgjQVgkpm8nhNEfDf6txHznoYREg9kirmJrkivgL4oSEimFF6nsQ6M41QvhM2Z33nves5vfSn9n1UwNFJBYtWVnHYMATn76vLuL3zU88KyeAYcHfsih3He6UHcXDxcaecHVz6jhCYz1P2UZn2bDVruL5wXpehgBfBaLKm3Ba","blskey_pop":"RahHYiCvoNCtPTrVtP7nMC5eTYrsUA8WjXbdhNc8debh1agE9bGiJxWBXYNFbnJXoXhWFMvyqhqhRoq737YQemH5ik9oL7R4NTTCz2LEZhkgLJzB3QRQqJyBNyv7acbdHrAT8nQ9UkLbaVL9NBpnWXBTw4LEMePaSHEw66RzPNdAX1","client_ip":"127.0.0.1","client_port":9702,"node_ip":"127.0.0.1","node_port":9701,"services":["VALIDATOR"]},"dest":"Gw6pDLhcBcoQesN72qfotTgFa7cbuqZpkX3Xo6pLhPhv"},"metadata":{"from":"Th7MpTaRZVRYnPiabds81Y"},"type":"0"},"txnMetadata":{"seqNo":1,"txnId":"fea82e10e894419fe2bea7d96296a6d46f50f93f9eeda954ec461b2ed2950b62"},"ver":"1"}
```
Comment (gmulhearn-anonyome) [inline uniffi_aries_vcx/demo/app/src/main/java/org/hyperledger/ariesvcx/HomeScreen.kt:36]: move this as a util function to a separate file

File: tools/libvcx_logger/src/lib.rs [hunk 3]
```diff
@@ -66,14 +66,14 @@ impl LibvcxDefaultLogger {
...
-                    _ => Filter::default().with_min_level(log::Level::Error),
+                    "error" => Config::default().with_max_level(log::LevelFilter::Error),
+                    "warn" => Config::default().with_max_level(log::LevelFilter::Warn),
+                    "info" => Config::default().with_max_level(log::LevelFilter::Info),
+                    "debug" => Config::default().with_max_level(log::LevelFilter::Debug),
+                    "trace" => Config::default().with_max_level(log::LevelFilter::Trace),
+                    _ => Config::default().with_max_level(log::LevelFilter::Error),
                 },
-                None => Filter::default().with_min_level(log::Level::Error),
+                None => Config::default().with_max_level(log::LevelFilter::Error),
```
Comment (gmulhearn) [inline tools/libvcx_logger/src/lib.rs:76]: nice find on this. thanks for taking me thru the change yesterday

File: uniffi_aries_vcx/core/Cargo.toml (hunk 1 above)
Comment (gmulhearn) [reply uniffi_aries_vcx/core/Cargo.toml:24]: @swaptr can it be removed?

**Review by gmulhearn — DISMISSED** [review]
good stuff. LGTM, just bogdan's comment. and also will need a rebase after this: https://github.com/hyperledger/aries-vcx/pull/1014

(altho we can rebase from the github PR UI)

File: uniffi_aries_vcx/core/Cargo.toml (hunk 1 above)
Comment (swaptr) [reply uniffi_aries_vcx/core/Cargo.toml:24]: Yes  we can safely remove it now.

**Review by gmulhearn — DISMISSED** [review]
nice, just that conflict to deal with

**Review by gmulhearn — APPROVED** [review]

**Pull request merged**